backend/src/packages/chaiNNer_pytorch/pytorch/processing/rife/IFNet_HDv3_v4_14_align.py

```diff
@@ -0,0 +1,187 @@
+
+# Original Rife Frame Interpolation by hzwer
+# https://github.com/megvii-research/ECCV2022-RIFE
+# https://github.com/hzwer/Practical-RIFE
+
+# Modifications to use Rife for Image Alignment by tepete/pifroggi ('Enhance Everything!' Discord Server)
+
+# Additional helpful github issues
+# https://github.com/megvii-research/ECCV2022-RIFE/issues/278
+# https://github.com/megvii-research/ECCV2022-RIFE/issues/344
+
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+import torchvision.transforms as transforms
+from .warplayer import warp
+# from train_log.refine import *
+#import logging
+
+# Setup logging
+#logging.basicConfig(filename='debug.log', level=logging.DEBUG, format='%(asctime)s %(levelname)s:%(message)s')
+
+
+#device = torch.device("cuda" if torch.cuda.is_available() else "cpu")

```

Comment:
can you get rid of all these random comments?

Reply:
Okay, should we remove the links/credits to the original code as well?